infrastructure/main.tf

```diff
@@ -95,8 +95,8 @@ resource "local_file" "mwaa_variables" {
       ecs_cluster_name        = module.mwaa.cluster_name
       log_group_name          = module.mwaa.log_group_name
       mwaa_execution_role_arn = module.mwaa.mwaa_role_arn
-      assume_role_read_arn    = var.assume_role_arns[0]
-      assume_role_write_arn   = var.assume_role_arns[0]
+      assume_role_read_arn    = length(var.assume_role_arns) > 0 ? var.assume_role_arns[0] : ""
+      assume_role_write_arn   = length(var.assume_role_arns) > 0 ? var.assume_role_arns[0] : ""
```

Comment:
so not `[0]` and `[1]` both `[0]`? just making sure

Reply:
we're using the same role for both read and right. But I think I should make it [1]. Let me push a change